x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/openai/OpenAiServiceTests.java

```diff
@@ -122,221 +145,184 @@ public void shutdown() throws IOException {
         webServer.close();
     }
 
-    public void testParseRequestConfig_CreatesAnOpenAiEmbeddingsModel() throws IOException {
```

Comment:
Since `OpenAiServiceTests` now extends `AbstractInferenceServiceTests` these tests are contained in the base class or in the parameterized variant.